Makefile

```diff
@@ -6,13 +6,15 @@ UGLIFY_BIN = ./node_modules/.bin/uglifyjs
 
 test:
 	@NODE_ENV=test $(MOCHA_BIN) \
+	  --exit \
```

Comment:
Mocha no longer exits by itself if there are things to clean up. Adding this exit option will mimic the older Mocha version we were using